SYCL/ESIMD/api/functional/ctors/ctor_default.cpp

```diff
@@ -54,7 +54,7 @@ struct var_declaration {
 
 // Descriptor class for the case of calling constructor in rvalue in an
 // expression context
-struct rval_in_expression {
+struct rval_in_express {
```

Comment:
```suggestion
struct rval_in_expr {
```

Reply:
I've reverted this comment [here](https://github.com/intel/llvm-test-suite/pull/588/commits/f4c3940c7788727dae45410460dde6655274266d#diff-e20a89556dfa9ec98f8eec4bea698de3975b780ff24a3bde1ced890af1d16581L60-R61)